kubectl-intermediate/step12.md

```diff
@@ -21,10 +21,10 @@ The previous command's output should now look similar to the below output:
 ```
 NAME        CPU
 master       2
-node01       4
+node01       2
 ```
 
-In step 9, we deployed multiple pods, so let's find all the pods that were deployed with the label `image:nginx:1.16` and output them in a tabulated format with column headers POD_NAME and IMAGE_VER:
+In step 9, we deployed multiple pods, so let's find all the pods that were deployed and output them in a tabulated format with column headers POD_NAME and IMAGE_VER:
```

Comment:
Instead of `label`, I believe this should say `image name`.

Reply:
the reason i removed the statement about the label was because, as you point out, it was somewhat erroneous or unclear.

Reply:
I'm sorry I read this backwards originally and thought it was added, my mistake.